src/test/resources/data/dimension_seed_test/dimension/fixed_seed_dimension.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "minecraft:overworld",
+  "generator": {
+    "biome_source": {
+      "preset": "minecraft:overworld",
+      "type": "minecraft:multi_noise"
+    },
+    "seed": 12345,
```

Comment:
Is it feasible to allow this flag to be datagenerated with other dimension jsons?

Reply:
* Correct me if I'm wrong here, but as far as I can tell I can't find any builtin datagenerators for dimensions/levelstems in vanilla or forge
* Adding a dimension datagenerator to forge wouldn't be complicated but it might be outside the scope of this PR
  * Anyone can make a datagenerator for any worldgen json by skooshing together RegistryWriteOps, a worldgen codec, and DataProvider.save(), not sure whether adding a helper method for that would be within the scope of forge or not
* Any dimension/levelstem datagenerator that *does* leverages codecs would automatically support this flag since the flag is being added to the codec

Reply:
I couldn't confirm the existence of a dimension datagenerator, so thanks for the clarification.